src/customize-pop/gnome-tweaks-extensions/gnome-extensions.md

```diff
@@ -0,0 +1,62 @@
+# GNOME Shell Extensions
+
+GNOME shell extensions are features written by third party developers that build upon the GNOME Shell. Extensions are similar to Chrome Extensions or Firefox Addons.
+
+- **GNOME Extensions** is an application for managing these extensions.
+- **Extension Manager** is a utility for browsing and installing extensions.
+
+Both applications provide menus for configuring, disabling, or removing installed extensions.
+
+---
+
+## Installing the GNOME Extensions & Extension Manager Apps
+
+Launch the Pop!\_Shop and type `extensions` then click `Install` for each application.
+
+![Install Extensions App](/images/gnome-tweaks-extensions/install-extensions-app.png)
+
+## Built-In Extensions
+
+Components packaged in Pop!\_OS are listed in the `Built-In` section. Toggle the switches to enable or disable an extension.
+
+![Built-in Ext](/images/gnome-tweaks-extensions/built-in.png)
+
+These features comprise the Computer Operating System Main Interface Components (COSMIC).
+
+| Extension | Description |
+|-----------|-------------|
+| [Cosmic Dock](https://github.com/pop-os/cosmic-dock) | A modified version of Dash to Dock with different defaults. |
+| [Cosmic Workspaces](https://github.com/pop-os/cosmic-workspaces) | Pop!\_OS's workspaces feature with vertical stacking. |
+| [Pop COSMIC](https://github.com/pop-os/cosmic) | The collection of unique components that comprise the Pop!\_OS experience: Customized Dash-to-Dock, vertical workspaces, and additional configuration options in Settings. |
```

Comment:
The COSMIC extension itself mainly provides the standalone Applications menu, in addition to tying some of the other extensions that you mentioned together with small tweaks.

Reply:
Thanks for clarifying, updated.